Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterCreateEditWidget.cpp

```diff
@@ -130,14 +121,38 @@ namespace EMStudio
         }
     }
 
-    void ParameterCreateEditDialog::Init()
+    void ParameterCreateEditWidget::Init(const EMotionFX::Parameter* editParameter)
     {
-        if (m_valueTypeCombo)
+        Reinit(editParameter);
+    }
+
+    void ParameterCreateEditWidget::Reinit(const EMotionFX::Parameter* editParameter)
+    {
+        bool showType = !editParameter || (editParameter && editParameter->RTTI_IsTypeOf(azrtti_typeid<EMotionFX::ValueParameter>()));
```

Comment:
Nit: I would actually put line 136 up in order before line 131, and follow the line 131-134 immediately with line 149. This way you are using the value you just created, which makes the comment below much more relevent.
Nit2: Add const (const bool showType =...)